tests/test_ensembles/hist_gradient_boosting/hist_gradient_boosting_regressor.py

```diff
@@ -0,0 +1,11 @@
+from sklearn.ensemble import HistGradientBoostingRegressor
+from pymilo.utils.test_pymilo import pymilo_regression_test
+from pymilo.utils.data_exporter import prepare_simple_regression_datasets
+
+MODEL_NAME = "HistGradientBoostingRegressor"
+
+def hist_gradient_boosting_regressor():
+    x_train, y_train, x_test, y_test = prepare_simple_regression_datasets()
+    hist_gradient_boosting_regressor = HistGradientBoostingRegressor().fit(x_train, y_train)
+    pymilo_regression_test(hist_gradient_boosting_regressor, MODEL_NAME, (x_test, y_test))
+
```

Comment:
remove this extra line.

Reply:
It raises an error, according to a standard, I have to add a new line there and it is there for every single model test we have so far.

Reply:
I found out why only these files are like this, it is because of some extra comments I've added to these files and didn't consider them when pushing up, now that I've updated the `CHANGELOG.md`, I removed them and it is fixed now.